### ISSUE TITLE
Video Sequencer - re-organize the Change Effect Type menu's to be consistent with Add menus

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#894]
## Issue
![PureRef_2019-05-18_20-56-44](https://user-images.githubusercontent.com/35357409/57976835-935b9080-79af-11e9-9db7-1049657ab0c4.png)
When you compare the Add menu's for effect strips in the VSE, you get a very different order and system when you want to "Change Effect Type", not only are transition strips grouped with the effect strips, the order is all out of whack. There are even some entries that aren't there! 

## Solution
Re-ordering, splitting up, or adding dividers to make it consistent would be great - so they are all the one and the same order for muscle memory and consistency. 

## Alternatives
Another solution could be to divide the Change Effect Type menu to remove the Transition strips to have it's own menu, similar to the add menus.

**Comment by ReinerBforartists** [comment]
Please wait with this changes a bit longer. There are changes at the VSE code incoming ...

**Comment by Draise14** [comment]
Ah right, sure. What are they adding to these? 
Just curious, how do you add line breaks to enumerated menu's? I've had a look (one is built in Python, the other an enumerated C menu).

**Comment by ReinerBforartists** [comment]
Line break is \n

**Comment by ReinerBforartists** [comment]
To quote the dev meeting notes from tomorrow: 

> Sequencer menus and sidebar panels have been reorganized for consistency and clarity. (Peter Fog, William Reynish)

**Comment by ReinerBforartists** [comment]
Flight we have green. The merge is in the master now ^^

**Comment by Draise14** [comment]
Oh great.

**Comment by Draise14** [comment]
![PureRef_2019-05-21_09-40-37](https://user-images.githubusercontent.com/35357409/58105700-8a83de00-7bac-11e9-9047-c5a001546945.png)

Ok.. the effect strip organization is now the same as the Change Effect Type, great. 

But now I see a huge inconsistency.. as you can change some effect strips into things that aren't even considered effects, eg. adjustment layer, text, color and transition strips. 

Trying to change these effects to these other ones creates errors if you do try change an effect strip, like they should be in their own enumerated menu to swap out their own kind (transition strips), or removed entirely. 

@ReinerBforartists , what would you do? 
Should I commit and close like this and make a new task with the issue of creating a new "Change Transitions" enumerated menu? Then add another task to remove the out of place "effects" content strips (are we sure we want to change an effect strip to a content strip as a feature?) 
What category is a Adjustment Layer anyway? Wouldn't that be an effect, not shown on it's own (as in, be in the enumerated menu for adding effects)? 
I also need to put in the right icons for Color and Adjustment Layers - also another task.

**Comment by ReinerBforartists** [comment]
Yes. Work in as small as possible steps as possible!

I don't know the workflow, you know i keep my hands away from VSE. So i cannot say what is better. But there is no need to have it 100% consistent. And when you are unsure then keep this part in the way the Blender devs did keep it.

**Comment by Draise14** [comment]
Ok, will do. 

I have finished this one. Will make new tasks for other things. I noticed one of my older tasks reverted with the last merge. 

![PureRef_2019-05-21_12-28-44](https://user-images.githubusercontent.com/35357409/58117511-5caa9380-7bc4-11e9-935f-542e97263aa6.png)

- Added separators to enumerated menu to be a little more consistent without changing features
- Added icons to enumerated menu
- fixed icons to match in enumerated menu for consistency
